dockermodelrunner/inference/api.go

```diff
@@ -0,0 +1,12 @@
+package inference
+
+// ExperimentalEndpointsPrefix is used to prefix all <paths.InferencePrefix> routes on the Docker
+// socket while they are still in their experimental stage. This prefix doesn't
+// apply to endpoints on model-runner.docker.internal.
+const ExperimentalEndpointsPrefix = "/exp/vDD4.40"
```

Comment:
question: Is this out of date? Using as is results in failures when used in URL(path...) e.g. `Status()` fails as 80 is not bound in localhost, not sure if I'm missing something?